synapse/rest/client/v1/admin.py

```diff
@@ -784,6 +784,31 @@ def on_GET(self, request, target_user_id):
         defer.returnValue((200, ret))
 
 
+class DeleteGroupAdminRestServlet(ClientV1RestServlet):
+    """Allows deleting of local groups
+    """
+    PATTERNS = client_path_patterns("/admin/delete_group/(?P<group_id>[^/]*)")
```

Comment:
given https://github.com/matrix-org/synapse/issues/4850, it would be good to consider putting this elsewhere.

OTOH there's something to be said for first adding an alternative path for all the existing admin APIs, and only then start adding new ones on the new path.

Reply:
Point. Though I don't really know what exactly we want to do there